documentation/asciidoc/computers/remote-access/ssh.adoc

```diff
@@ -0,0 +1,176 @@
+. Select the *Allow public-key authentication only* radio button. If you already have an SSH public key stored in `~/.ssh/id_rsa.pub`, Imager automatically uses that public key to prefill the text box. If Imager doesn't find an SSH public key, you can click the *RUN SSH-KEYGEN* button to generate a new keypair.
+
+==== Manually configure an SSH key
+
+If you already have an installation of Raspberry Pi OS, you can update your existing configuration to use SSH key authentication.
+
+==== Check for existing SSH public keys
+
+To check for an existing SSH public key on the computer you use to remotely connect to the Raspberry Pi, run the following command:
+
+[source,console]
+----
+$ ls ~/.ssh
+----
+
+If you see files named `id_ed25519.pub`, `id_rsa.pub`, or `id_dsa.pub`, you already have an SSH key. Skip SSH keypair generation and proceed to xref:remote-access.adoc#add-ssh-key-identity[add the SSH key to your list of SSH identities].
+
+==== Generate new SSH keypair
+
+TIP: This guide provides instructions to generate a new RSA key. For additional security, you can instead generate a http://ed25519.cr.yp.to/[Ed25519] key. Replace `rsa` with `ed25519` in your public and private key file names and the instructions should work the same.
+
+To generate a new SSH keypair, enter the following command:
+
+[source,console]
+----
+$ ssh-keygen
+----
+
+When asked where to save the key, press *Enter* to use the default location, `~/.ssh/id_rsa`.
+
+When asked for an optional keyphrase, press *Enter* to use no keyphrase.
+
+Run the following command to check the contents of the `.ssh` directory:
+
+[source,console]
+----
+$ ls ~/.ssh
+----
+
+You should see the files `id_rsa` and `id_rsa.pub`:
+
+----
+authorized_keys  id_rsa  id_rsa.pub  known_hosts
+----
+
+The `id_rsa` file contains your private key. Keep this secure on the computer you use to remotely connect to the Raspberry Pi.
+
+The `id_rsa.pub` file contains your public key. You will share this key with your Raspberry Pi. When you connect with the Raspberry Pi remotely, it will use this key to verify your identity.
+
+[[add-ssh-key-identity]]
+==== Add the SSH key to your list of SSH identities
+
+Start the SSH agent:
+
+[source,console]
+----
+$ eval "$(ssh-agent -s)"
+----
+
+Next, add your key identities to `ssh-agent` with the following command:
+
+[source,console]
+----
+$ ssh-add ~/.ssh/id_rsa
+----
+
+[[copy-your-public-key-to-your-raspberry-pi]]
+==== Copy the public key to your Raspberry Pi
+
+On the computer you use to remotely connect to the Raspberry Pi, use the following command to securely copy your public key to the Raspberry Pi:
+
+[source,console]
+----
+$ ssh-copy-id <username>@<ip address>
+----
+
+When prompted, enter the password for your user account on the Raspberry Pi.
+
+TIP: If your operating system does not support `ssh-copy-id`, you can copy your public key with xref:remote-access.adoc#scp[`scp`] or a network share instead.
```

Comment:
This TIP would probably be more useful if it said _where_ to copy the public key to? :wink: